README.md

```diff
@@ -1,22 +1,85 @@
-# cluster-logging-operator
+# Cluster Logging Operator
+
 An operator to support OKD aggregated cluster logging.  Cluster logging configuration information
 is found in the [configuration](./docs/configuration.md) documentation.
 
-## Quick Start
+# Hacking
+
+Note: In the rest of this doc "CLO" stands for Cluster Logging Operator
+
+## Preparation
+
+Clone this repository into $GOPATH/github.com/openshift/cluster-logging-operator
+
+Install `podman` and `skopeo`. For example on  Fedora:
+```
+sudo dnf install -y podman skopeo
+```
 
-To get started with cluster logging and the `cluster-logging-operator`:
-* Ensure `docker` is installed on your local system [(**Note** for Fedora 31)](./docs/fedora31.md)
-* Ensure `skopeo` is installed, e.g. `sudo dnf install -y skopeo`
+Check out the other required repositories in your $GOPATH:
 ```
-$ oc login $CLUSTER -u $ADMIN_USER -p $ADMIN_PASSWD
-$ REMOTE_CLUSTER=true make deploy-example
+go get -d -u github.com/openshift/elasticsearch-operator/... github.com/openshift/origin-aggregated-logging/...
 ```
-This will stand up a cluster logging stack named 'example'.
 
+## Testing
+
+The unit tests can be run without a cluster:
+* `make`: Run unit tests, lint and format code.
+* `make test-unit`: Just run unit tests.
+
+All remaining tests require a cluster.
+
+You can build and deploy the CLO image into the cluster:
+* `make deploy`: Build CLO image and deploy to cluster with elasticsearch.
+* `make undeploy`: Undo `make deploy`.
+
+Alternatively you can run it as a local process (see below)
+* `make run`: Run CLO as a local process. Does not require `make deploy`.
 
-## Hacking
+After running  either `make deploy` or `make run`, you can run non-unit tests:
+* `make test-cluster` Run non-unit tests against an existing CLO.
+* `go run test/e2e/something`: Run tests individually.
 
-### Building a Universal Base Image (UBI) based image
+Some other useful targets (read the Makefile for more) 
+* `make image`: Build the image as $IMAGE_TAG, do not deploy it.
+* `make test-olm-e2e`: Run e2e the way the CI server does.
```

Comment:
We should mention `make deploy-image` here.

Reply:
I'll improve the README with this and your earlier comments